boards/arm/nucleo_f446re/doc/index.rst

```diff
@@ -223,3 +235,6 @@ You can debug an application in the usual way.  Here is an example for the
 
 .. _STM32F446 reference manual:
    http://www.st.com/resource/en/reference_manual/dm00135183.pdf
+
+.. _RS485 CAN Shield:
```

Comment:
I think we should not link to any shields, but rather say something like
"Connect to a CAN transceiver"

Reply:
Ahh, that shield is made for the nucleo. Sry, forget the comment before.